src/shared/lib/pathwayMapper/PathwayMapperTable.tsx

```diff
@@ -82,8 +83,13 @@ export default class PathwayMapperTable extends React.Component<
         this._columns = {};
         this.generateColumns();
     }
-
+    private sc_column = '';
     generateColumns() {
+        //this condition is used to set the label of the score column
+        this.sc_column = 'Score';
+        if (this.props.view === 'patient') {
+            this.sc_column = '# Genes matched';
+        }
```

Comment:
This is not ideal, we shouldn't introduce conditions based on hardcoded views. Instead we should use something like `columnsOverride`